graylog2-web-interface/src/components/common/SortableList.jsx

```diff
@@ -68,9 +92,9 @@ class SortableList extends React.Component {
     });
 
     return (
-      <ListGroup className={this.props.disableDragging ? 'sortable-list' : 'sortable-list sortable-list-cursor'}>
+      <SortableListGroup disableDragging={this.props.disableDragging}>
```

Comment:
It looks like this is missing some margins.

Master:
<img width="613" alt="Screenshot 2020-02-26 at 17 25 41" src="https://user-images.githubusercontent.com/716185/75365191-202b6700-58bd-11ea-804b-4a87710398b5.png">

This branch:
<img width="614" alt="Screenshot 2020-02-26 at 17 25 09" src="https://user-images.githubusercontent.com/716185/75365206-24578480-58bd-11ea-8490-37383ed14cfc.png">

Reply:
Nice catch, also appears here 
![image](https://user-images.githubusercontent.com/122591/75393194-21f02d00-58b3-11ea-9fad-99f0b65396e6.png)

Reply:
![image](https://user-images.githubusercontent.com/122591/75394304-798f9800-58b5-11ea-8558-2bdc7e825770.png)